### ISSUE TITLE
N-Triples parser breaks on colons in blank node names

### ISSUE THREAD

**Issue by wouterbeek** [SWI-Prolog/packages-semweb#35]
The N-Triples parser breaks whenever a colon appear after the `_:'-prefix inside a blank node name.  Example document:

```ntriples
_:: <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Resource> .
```

The relevant N-Triples 1.1 grammar rules are:

```bnf
[141s] 	BLANK_NODE_LABEL 	::= 	'_:' (PN_CHARS_U | [0-9]) ((PN_CHARS | '.')* PN_CHARS)?
[158s] 	PN_CHARS_U 	::= 	PN_CHARS_BASE | '_' | ':'
```

**Comment by JanWielemaker** [comment]
!?  I see 

    [164s] 	PN_CHARS_U 	::= 	PN_CHARS_BASE | '_'

in https://www.w3.org/TR/turtle/#grammar-production-PN_CHARS_U

Did you pick the wrong version or am I looking at the wrong version.  Surely, allowing for a : in a blank node label seems an odd choice to me.

**Comment by wouterbeek** [comment]
Hi Jan,

You are referring to the Turtle 1.1 grammar.  The colon is only allowed in [N-Triples 1.1](https://www.w3.org/TR/n-triples/#n-triples-grammar) and [N-Quads 1.1](https://www.w3.org/TR/n-quads/#sec-grammar).

PS: I know what you mean though: we're always told that N-Triples and N-Quads are subsets of Turtle, but this is not exactly the case, if only because some N-Triples/N-Quads blank node labels are not Turtle blank node labels.

**Comment by JanWielemaker** [comment]
Oh well ...  Fixed with f58968eff88af5f29df0563f948dcb408d0ed861
